master/usage/openstack/labels.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Endpoint Labels
+canonical_url: 'https://docs.projectcalico.org/v3.2/usage/openstack/labels'
+---
+
+When {{site.prodname}} represents an OpenStack VM as a Calico WorkloadEndpoint, it puts
+labels on the WorkloadEndpoint to identify the project and security groups that the VM
+belongs to.
+
+For the VM's OpenStack project (previously known as 'tenant'), those labels are:
+
+| Label Name                               | Value                   |
+|------------------------------------------|-------------------------|
+| projectcalico.org/openstack-project-id   | <the VM's project ID>   |
+| projectcalico.org/openstack-project-name | <the VM's project name> |
+|------------------------------------------|-------------------------|
+
+For each security group that the VM belongs to, those labels are:
+
+| Label Name                                                  | Value                   |
+|-------------------------------------------------------------|-------------------------|
+| sg.projectcalico.org/openstack-\<security group ID\>        | \<security group name\> |
```

Comment:
I've seen backticks used more often than escaping  for this sort of thing.

Reply:
Do you mean like this:

sg.projectcalico.org/openstack-`<security group ID>`

Reply:
I am not sure that backticks really escape anything, but they do signify that the enclosed text should be displayed in a code format. Because it's treated as code, control characters inside the text are effectively escaped. You could enclose the entire label in backticks and then you would not need these `\` characters to escape the `>` and `<` characters. 

However, you could also just leave the `\` since these are the true escape characters.

I would recommend enclosing the entire string in backticks as it seems like code to me.